docs/tutorials/tutorial_experience.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>Maybe we could link to the basic Robinson Crusoe Tutorial here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>